test/Magick.rb

```diff
@@ -79,8 +79,8 @@ def test_fonts
       assert_instance_of(String, f.name)
       assert_instance_of(String, f.description) unless f.description.nil?
       assert_instance_of(String, f.family)
-      assert_instance_of(Magick::StyleType, f.style)
-      assert_instance_of(Magick::StretchType, f.stretch)
+      assert_instance_of(Magick::StyleType, f.style) unless f.style.nil?
```

Comment:
Hmm, it feels weird to have this sort of conditional in the tests. Do we know *when* it will be `nil`? Maybe we can have different tests for the cases where it is or isn't `nil`.